internal/controller/apiextensions/offered/reconciler.go

```diff
@@ -42,7 +43,9 @@ import (
 	"github.com/crossplane/crossplane-runtime/pkg/resource/unstructured"
 
 	v1 "github.com/crossplane/crossplane/apis/apiextensions/v1"
+	"github.com/crossplane/crossplane/apis/secrets/v1alpha1"
```

Comment:
Might be worth importing this as `secretsv1alpha1` or similar, just to make the references down below more obvious?